sunpy/__init__.py

```diff
@@ -48,4 +48,23 @@ class UnsupportedPythonError(Exception):
     # Load user configuration
     config = load_config()
 
+    import logging
+
+    # Use the root logger as a dummy log before initializing Astropy's logger
+    log = logging.getLogger()
+
+    from astropy.logger import _init_log
+
+    #  read and translate settings for Astropy's logger
+    if config.has_section('logger'):
+        from astropy.logger import Conf as LoggerConf
+        conf = LoggerConf()
+        option_list = ['log_level', 'use_color', 'log_warnings', 'log_exceptions', 'log_to_file', 'log_file_path',
```

Comment:
What happens if someone has a config file but not all these options? Do they default sensibly or error?

Reply:
The LoggerConf has default values already set. All this does is update the ones that the user defined.